docs/editor/integrated-terminal.md

```diff
@@ -24,31 +24,70 @@ To open the terminal:
-"terminal.integrated.shell.windows":"C:\\Windows\\system32\\WindowsPowerShell\\v1.0\\powershell.exe"
+// 64-bit cmd it available, otherwise 32-bit
+"terminal.integrated.shell.windows":"C:\\Windows\\sysnative\\cmd.exe"
+// 64-bit PowerShell if available, otherwise 32-bit
+"terminal.integrated.shell.windows":"C:\\Windows\\sysnative\\WindowsPowerShell\\v1.0\\powershell.exe"
+// Git Bash
+"terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe"
+// Bash on Ubuntu (on Windows)
+"terminal.integrated.shell.windows": "C:\\Windows\\sysnative\\bash.exe"
 ```
 
-Arguments can be passed to the terminal shell on Linux and OS X using the `terminal.integrated.shellArgs.*` settings.
+### Linux & OS X
+
+Typically `$SHELL` is your primary shell on Unix-like systems so you probably won't need to change the shell. Passing arguments to the shell may be done however, enabling for example running bash as a login shell (which runs `.bash_profile`) by passing in the `-l` argument.
+
+```json
+// Linux
+"terminal.integrated.shellArgs.linux": ["-l"]
+// OS X
+"terminal.integrated.shellArgs.osx": ["-l"]
+```
 
 ## Key Bindings
 
 The **Toggle Integrated Terminal** command is bound to `kb(workbench.action.terminal.toggleTerminal)` to quickly toggle the integrated terminal panel in and out of view.
 
-Other terminal commands are available and can be bound to your preferred key board shortcuts.
+Other terminal commands are available and can be bound to your preferred keyboard shortcuts.
 
 They are:
 
 * `workbench.action.terminal.focus`: Focus the terminal. This is like toggle but focuses the terminal instead of hides it if its visible.
 * `workbench.action.terminal.focusNext`: Focuses the next terminal instance.
 * `workbench.action.terminal.focusPrevious`: Focuses the previous terminal instance.
 * `workbench.action.terminal.kill`: Remove the current terminal instance.
+* `workbench.action.terminal.runSelectedText`: Runs the selected text in the termianl instance
+
+### Copy & Paste
+
+#### OS X
+
+Copy and paste on OS X can be done using the standard keys, `kbstyle(cmd+c)` and `kbstyle(cmd+v)` respectively.
+
+#### Linux & Windows
+
+Copy and paste on Linux & Windows can be done using `kbstyle(ctrl+ins)` and `kbstyle(shift+ins)` respectively. 
+
+> Pre-release: This is changing in the upcoming version to `kbstyle(ctrl+shift+c)` and `kbstyle(ctrl+shift+v)` respectively, this change is live in the [Insiders build](https://code.visualstudio.com/insiders).
 
 ## Managing Multiple Terminals
 
 You can create multiple terminals (`+` button) open to different locations and easily navigate between them.
 
 ![Multiple Terminals](images/integrated-terminal/terminal-multiple-instances.png)
+
+If you plan on utilizing multiple terminals extensively it would be a good idea to add keybindings for the `focusNext`, `focusPrevious` and `kill` commands outlined in the [Key Bindings section](#_key-bindings) to allow navigation between them using only the keyboard.

```

Comment:
Use the full path so that the link will work on GitHub.
(/docs/editor/integrated-terminal.md#key-bindings)